CredScanSuppressions.json

```diff
@@ -2,8 +2,16 @@
 	"tool": "Credential Scanner",
 	"suppressions": [
 		{
-			"file": "server/routerlicious/packages/routerlicious/config/telegraf/telegraf.conf",
-			"_justification": "Sample username password placeholders as part of code comments"
```

Comment:
Removed this file from the list of suppressions because this PR also updates the dummy credentials in it to say PLACEHOLDER, which the CredScan tool understands is not a leaked value.